sr_utilities/scripts/interactive_marker.py

```diff
@@ -130,5 +130,6 @@ def processFeedback(self, feedback):
 if __name__ == "__main__":
     rospy.init_node("simple_marker")
 
-    int_mark = InteractiveConnectorSelector(["srh/position/palm"], None)
+    # Error here, file probably very depricated
+    int_mark = InteractiveConnectorSelector(["srh/position/palm"], None)  # pylint: disable=E1120
```

Comment:
The constructor is def __init__(self, object_names, callback_fct, interactive_server), so 3 args are needed? 
There's a comment above main about it being used for debug, so maybe the main can be removed?

Reply:
I'm not sure if anyone is ever gonna try and use this file again, but if they do then maybe `["srh/position/palm"]` as the `object_names` would be a useful hint?

Reply:
Agreed, my preferred solution is to remove `main` and this file from installed python executables.

Reply:
main removed here 55a6f2f7c523c3c3b14b1771ed976b52db5199b5

Reply:
removed from install python executables here b003c18c2a69cd84f105ddd699ab596fda1574c9